src/main/java/core/basesyntax/service/calculationservice/FruitService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.calculationservice;
+
+public interface FruitService {
+    String calculationFruits(String nameFruit);
```

Comment:
not fixed
https://github.com/mate-academy/jv-fruit-shop/pull/657#discussion_r1102948963